pyEpiabm/pyEpiabm/core/place.py

```diff
@@ -59,6 +65,15 @@ def set_max_cap(self, max_capacity: int):
         """
         self.max_capacity = max_capacity
 
+    def set_mean_cap(self, mean_capacity: float):
+        """Sets the maximum capacity of a place.
+
+        :param mean_capacity: Maximum number of people
+            allowed in place
+        :type mean_capacity: float
```

Comment:
```suggestion
        :param mean_capacity: Mean capacity of place type
        :type mean_capacity: float
```
Not sure if this matches its use, but do correct the docstring either way